src/appleseed-max-impl/oslutils.cpp

```diff
@@ -224,8 +224,13 @@ void connect_float_texture(
     const char*         material_node_name,
     const char*         material_input_name,
     Texmap*             texmap,
-    const float         multiplier)
+    const float         const_value)
 {
+    auto time = GetCOREInterface()->GetTime();
+    float output_amount = 1.0f;
```

Comment:
Alternatives allowing to make `output_amount` const:

    const float output_amount = texmap != nullptr ? get_output_amount(texmap, time) : 1.0f;

or

    const float output_amount =
        texmap != nullptr
            ? get_output_amount(texmap, time)
            : 1.0f;